frontend/src/App.jsx

```diff
@@ -120,7 +121,15 @@ const router = createBrowserRouter([
       {
         path: appRoutes.EDIT_USER,
         element: <EditUser />
-      }
+      },
+      // TODO: Modify OrganizationLayout to support child route integration, then use it for the following route.
```

Comment:
Routes on our release-0.2.0 branch have been updated to account for this. Please either merge or rebase on top of this branch to get latest changes.

Reply:
Sure! Thanks for the information.